libkernelflinger/security_efi.c

```diff
@@ -188,15 +227,25 @@ BOOLEAN is_eom_and_secureboot_enabled(VOID)
 	return sbflags && enduser;
 }
 
+static void dump_seed(__attribute__((unused)) UINT8 *seed)
+{
+#if 1  // Change to 1 for debug the RPMB keys
+	CHAR16 buf[TRUSTY_SEED_SIZE * 2 + 2];
```

Comment:
This should be:
`CHAR16 buf[TRUSTY_SEED_SIZE * sizeof(CHAR16) + sizeof(CHAR16)];`